rstudio/rstudio
# update xterm.js to ver 4.11.0

### Intent

Update terminal emulator library (xterm.js) used by IDE to latest version (4.11.0) to pick up bug fixes.

Addresses #9195

### Approach

Mechanical, update version in script and run the script.

### Automated Tests

No new tests needed.

### QA Notes

Sanity check that terminal is still working in the IDE ("working" meaning you can start a terminal, type into it, and see what you expect). Desktop or Server is fine, doesn't matter.

If you want more details, you can check the list of fixes in release notes for xterm.js 4.11, and 4.10 (we were previously using 4.9). https://github.com/xtermjs/xterm.js/releases

### Checklist
